app/assets/javascripts/snowcrash/modules/auto_save.js

```diff
@@ -0,0 +1,49 @@
+document.addEventListener( "turbolinks:load", function(){
+  console.log('auto_save loaded');
+
+  document.querySelectorAll("[data-behavior~=auto-save]").forEach( function(item){
```

Comment:
Why not use jquery selector?

```suggestion
  $("[data-behavior~=auto-save]").each( function(index){
```
then use `$(this)` for the item.

Reply:
because I can see a future in which we're going to move away from jQuery... I think that if we keep this in mind when adding little snippets, we'll have an easier time when it's time to pull the plug.